packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/main_executable_resilient/TestMainExecutableResilient.py

```diff
@@ -0,0 +1,86 @@
+# TestMainExecutableResilient.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2019 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+"""
+Test `@_implementationOnly import` in the main executable with a resilient library
+"""
+import commands
+import lldb
+from lldbsuite.test.lldbtest import *
+import lldbsuite.test.decorators as decorators
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import os.path
+import time
+import unittest2
+
+class TestMainExecutable(TestBase):
+
+    mydir = TestBase.compute_mydir(__file__)
+
+    def launch_and_break(self):
+        exe_name = "main"
```

Comment:
See above.

Reply:
Yeah, this whole directory is identical to the other one but with `-enable-library-evolution` added to the Makefile.